docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,189 @@
+---
+id: troubleshooting
+title: Troubleshooting OpenEBS - Overview
+---
+
+### General guidelines for troubleshooting
+
+- Search for similar issues mentioned in this page as well as the following troubleshooting sections.
+  - [Troubleshooting Install](/docs/troubleshooting/t-install).
+  - [Troubleshooting Uninstall](/docs/troubleshooting/t-uninstall).
+  - [Troubleshooting NDM](/docs/troubleshooting/t-ndm).
+  - [Troubleshooting Jiva](/docs/troubleshooting/t-jiva).
+  - [Troubleshooting cStor](/docs/troubleshooting/t-cstor).
+  - [Troubleshooting LocalPV](/docs/troubleshooting/t-localpv).
+- Contact [OpenEBS Community](/docs/introduction/community) for support.
+- Search for similar issues on [OpenEBS GitHub repository](https://github.com/openebs/openebs/issues).
+- Search for any reported issues on [StackOverflow under OpenEBS tag](https://stackoverflow.com/questions/tagged/openebs).
+
+#### Kubernetes related
+
+[Kubernetes node reboots because of increase in memory consumed by Kubelet](#node-reboot-when-kubelet-memory-increases)
+
+[Application and OpenEBS pods terminate/restart under heavy I/O load](#Pods-restart-terminate-when-heavy-load)
+
+#### Others
+
+[Nodes in the cluster reboots frequently almost everyday in openSUSE CaaS](#reboot-cluster-nodes)
+
+<font size="6" color="green">Kubernetes related</font>
```

Comment:
Use markdown heading here

Reply:
In markdown, we cannot color the heading using `###`. For that HTML code will be used, so retained the previous code